### ISSUE TITLE
Automatic Profiling Setup for Sentry in Laravel Forge

### ISSUE THREAD

**Issue by dalnoki** [getsentry/sentry#78682]
### Problem Statement

Currently, setting up profiling in Sentry with Laravel Forge requires manual intervention by accessing the console and installing packages like `php-excimer` via apt-get. This defeats one of the core benefits of Forge, which is to simplify server management by avoiding the need to use the command line for such tasks. 

For example, Forge automatically detects the presence of Octane, and if it's available, there's a little toggle to enable certain functions. There's currently no similar functionality to handle the installation `php-excimer` for profiling.

Basically, when a user configures Sentry and enables profiling, Forge should detect this need and check if php-excimer is installed. If php-excimer is not already installed, Forge should handle the installation automatically using apt-get install php-excimer, without requiring the user to access the console.

### Product Area

Profiling

**Comment by cleptric** [comment]
This is a bit out of our control, as we can't change how Forge works. I forwarded this to their team though 🙂
